vkcs/cdn/resource_resource/resource_resource_gen.go

```diff
@@ -0,0 +1,10085 @@
+						Description:         "Cache settings for users browsers.\nCache expiration time is applied to the following response codes: 200, 201, 204, 206, 301, 302, 303, 304, 307, 308. Responses with other codes will not be cached.",
+						MarkdownDescription: "Cache settings for users browsers.\nCache expiration time is applied to the following response codes: 200, 201, 204, 206, 301, 302, 303, 304, 307, 308. Responses with other codes will not be cached.",
+						PlanModifiers: []planmodifier.Object{
+							resource_planmodifiers.ResourceOption(),
+						},
+					},
+					"cors": schema.SingleNestedAttribute{
+						Attributes: map[string]schema.Attribute{
+							"enabled": schema.BoolAttribute{
+								Optional:            true,
+								Computed:            true,
+								Description:         "Controls the option state.",
+								MarkdownDescription: "Controls the option state.",
+							},
+							"value": schema.ListAttribute{
+								ElementType:         types.StringType,
+								Optional:            true,
+								Computed:            true,
+								Description:         "Value of the Access-Control-Allow-Origin header.\nPossible values:\n* [\"*\"] - adds \"*\" as the header value, content will be uploaded for requests from any domain.\n*[\"domain.com\", \"second.dom.com\"] - adds \"$http_origin\" as the header value if the origin matches one of the listed domains, content will be uploaded only for requests from the domains specified in the field.\n* [\"$http_origin\"] - adds \"$http_origin\" as the header value, content will be uploaded for requests from any domain, and the domain from which the request was sent will be added to the header in the response.",
+								MarkdownDescription: "Value of the Access-Control-Allow-Origin header.\nPossible values:\n* [\"*\"] - adds \"*\" as the header value, content will be uploaded for requests from any domain.\n*[\"domain.com\", \"second.dom.com\"] - adds \"$http_origin\" as the header value if the origin matches one of the listed domains, content will be uploaded only for requests from the domains specified in the field.\n* [\"$http_origin\"] - adds \"$http_origin\" as the header value, content will be uploaded for requests from any domain, and the domain from which the request was sent will be added to the header in the response.",
+							},
+						},
+						CustomType: CorsType{
+							ObjectType: types.ObjectType{
+								AttrTypes: CorsValue{}.AttributeTypes(ctx),
+							},
+						},
+						Optional:            true,
+						Computed:            true,
+						Description:         "Enables or disables CORS (Cross-Origin Resource Sharing) header support.\nCORS header support allows the CDN to add the Access-Control-Allow-Origin header to a response to a browser.",
+						MarkdownDescription: "Enables or disables CORS (Cross-Origin Resource Sharing) header support.\nCORS header support allows the CDN to add the Access-Control-Allow-Origin header to a response to a browser.",
```

Comment:
if \n is a part of markdown syntax?

Reply:
I believe `MarkdownDescription` is just meant for docs generators like our one, and in schema it actually replaces `Description` field, so they have the same meaning. But in the docs they say that provider developers should set both fields, as in the future this may change.